packages/app/src/debug/DebugContainer.vue

```diff
@@ -22,19 +22,25 @@
       />
       <div
         v-else-if="run?.status"
-        class="flex flex-col h-full"
+        class="flex flex-col h-full p-1.5rem gap-24px"
       >
+        <DebugNewRelevantRunBar
+          v-if="nextRelevantRun"
+          :gql="nextRelevantRun"
+          :current-run-number="run.runNumber"
+        />
         <DebugPageHeader
           :gql="run"
           :commits-ahead="props.commitsAhead"
         />
+        <TransitionQuickFade>
+          <DebugTestingProgress v-if="isRunning" />
+        </TransitionQuickFade>
+
         <DebugPendingRunSplash
-          v-if="isFirstPendingRun"
+          v-if="isRunning && (!run.totalFailed || run.totalFailed === 0)"
```

Comment:
I think this is overkill - wouldn't `!run.totalFailed` cover the `0` case as well?

Reply:
Yes it would.  Fixed